peers/validators/canonical_validator_client.go

```diff
@@ -98,77 +91,14 @@ func (v *CanonicalValidatorClient) GetValidatorSet(
 ) (map[ids.NodeID]*validators.GetValidatorOutput, error) {
```

Comment:
Should the comment for this function be updated to remove the sentence about falling back to getCurrentValidators?